persistence-jdbc/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/jdbc/JdbcPersistenceModuleSpec.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.persistence.jdbc
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import com.lightbend.lagom.internal.javadsl.api.broker.TopicFactory
+import com.lightbend.lagom.javadsl.api.ServiceLocator
+import com.typesafe.config.ConfigFactory
+import javax.inject.Inject
+import org.scalatest.{ BeforeAndAfterAll, Matchers, WordSpec, WordSpecLike }
+import play.api.Play
+import play.api.db.DBApi
+import play.api.inject.{ ApplicationLifecycle, DefaultApplicationLifecycle }
+import play.inject.guice.GuiceApplicationBuilder
+import play.api.inject.{
+  ApplicationLifecycle,
+  BindingKey,
+  DefaultApplicationLifecycle,
+  bind => sBind
+}
+
+/**
+ * This test will simply wire a minimal application using Guice.
+ *
+ * The LagomDbApiProvider is a copy from the Play that does not try to connect to the DB
+ * when created.
+ *
+ * In this particular test, we explicitly set a bogus DB configuration and let Guice create the class
+ * and wire then together. Because the Lagom impl does not connect eagerly no error should be thrown
+ * after wiring and the test should just pass.
+ */
+class JdbcPersistenceModuleSpec
```

Comment:
I tried to make a similar test for the scaladsl but its much more complicated.

In Scala, the actor system is being initialized and lots of other errors pop-up. I need to search further.